graphic_cheatsheet_group13.Rmd

```diff
@@ -0,0 +1,105 @@
+---
```

Comment:
please follow the guidelines [here](https://jtr13.github.io/cc22spring/github-submission-instructions.html), including remove the yaml header, add hash tags only for title and subtitles, add author name, and etc.